src/net_processing.cpp

```diff
@@ -4519,24 +4522,24 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
 
         auto& tx_process_time = state.m_tx_download.m_tx_process_time;
         while (!tx_process_time.empty() && tx_process_time.begin()->first <= current_time && state.m_tx_download.m_tx_in_flight.size() < MAX_PEER_TX_IN_FLIGHT) {
-            const uint256 txid = tx_process_time.begin()->second;
+            const GenTxid txid = tx_process_time.begin()->second;
```

Comment:
Not a super-strong opinion, but I'm wondering if we should call it `gtxid` or `gentxid` everywhere. txid used to mean something different before, and it still sort of does?

Reply:
Seems reasonable; done. I've changed all `GenTxid` variables in net_processing to be `gtxid`.